src/app/components/media/MediaFactCheck.js

```diff
@@ -188,81 +197,126 @@ const MediaFactCheck = ({ projectMedia }) => {
+              error={error}
+              autoGrow
+              placeholder={placeholder}
+              disabled={(!hasPermission || isDisabled)}
+              onBlur={(e) => {
+                const newValue = e.target.value;
+                setSummary(newValue);
+                handleBlur('summary', newValue);
+              }}
+            />
+          )}
+        </FormattedMessage>
+      </div>
+      <div className={inputStyles['form-fieldset-field']}>
+        <FormattedMessage
+          id="mediaFactCheck.urlPlaceholder"
+          defaultMessage="Add a URL to this fact-check article"
+          description="Placeholder instructions for fact-check URL field"
+        >
+          { placeholder => (
+            <TextField
+              label={<FormattedMessage id="mediaFactCheck.url" defaultMessage="Article URL" description="Label for fact-check URL field" />}
+              placeholder={placeholder}
+              defaultValue={url}
+              componentProps={{
+                id: 'media-fact-check__url',
+              }}
+              className="media-fact-check__url"
+              disabled={(!hasPermission || isDisabled)}
+              key={`media-fact-check__url-${claimDescription?.description ? '-with-claim' : '-no-claim'}`}
+              onBlur={(e) => {
+                const newValue = e.target.value;
+                let newUrl = newValue;
+                if (!/^https?:\/\//.test(newValue) && newValue && newValue.length > 0) {
+                  newUrl = `https://${newValue}`;
+                }
+                setUrl(newUrl);
+                handleBlur('url', newUrl);
+              }}
+            />
+          )}
+        </FormattedMessage>
+      </div>
       { languages.length > 1 ?
-        <Box my={2} >
+        <div className={inputStyles['form-fieldset-field']}>
           <LanguagePickerSelect
             isDisabled={(!hasPermission || isDisabled)}
             selectedLanguage={language}
             onSubmit={handleLanguageSubmit}
             languages={safelyParseJSON(team.get_languages)}
           />
-        </Box> : null
+        </div> : null
       }
 
       { projectMedia.team.smooch_bot ?
-        <Box mt={1}>
+        <div className={inputStyles['form-fieldset-field']}>
           <ButtonMain
             onClick={handleGoToReport}
             className="media-fact-check__report-designer"
             variant="contained"
             theme={published ? 'brand' : 'alert'}
             size="default"
             iconLeft={published ? <IconReport /> : <IconUnpublishedReport />}
-            disabled={saving || readOnly}
+            disabled={saving || readOnly || (!claimDescription || claimDescription.description?.trim()?.length === 0)}
```

Comment:
and reuse that variable named `claimDescriptionMissing` here? :D

Reply:
Great idea @amoedoamorim I have addressed this. Thanks for the great review overall